src/spawn_players.c

```diff
@@ -1283,53 +1203,25 @@ void func_8003D080(void) {
 
     switch (gActiveScreenMode) {
         case SCREEN_MODE_1P:
-            func_8003CD98(gPlayerOneCopy, camera1, 0, 0); // sic
-            func_8003CD98(gPlayerTwo, camera1, 1, 0);
-            func_8003CD98(gPlayerThree, camera1, 2, 0);
-            func_8003CD98(gPlayerFour, camera1, 3, 0);
-            func_8003CD98(gPlayerFive, camera1, 4, 0);
-            func_8003CD98(gPlayerSix, camera1, 5, 0);
-            func_8003CD98(gPlayerSeven, camera1, 6, 0);
-            func_8003CD98(gPlayerEight, camera1, 7, 0);
+            for (int i = 0; i < NUM_PLAYERS; i++) {
+                func_8003CD98(&gPlayers[i], camera1, i, 0); // sic
+            }
             break;
 
         case SCREEN_MODE_2P_SPLITSCREEN_HORIZONTAL:
         case SCREEN_MODE_2P_SPLITSCREEN_VERTICAL:
-            func_8003CD98(gPlayerOneCopy, camera1, 0, 0);
-            func_8003CD98(gPlayerTwo, camera1, 1, 0);
-            func_8003CD98(gPlayerThree, camera1, 2, 0);
-            func_8003CD98(gPlayerFour, camera1, 3, 0);
-            func_8003CD98(gPlayerFive, camera1, 4, 0);
-            func_8003CD98(gPlayerSix, camera1, 5, 0);
-            func_8003CD98(gPlayerSeven, camera1, 6, 0);
-            func_8003CD98(gPlayerEight, camera1, 7, 0);
-            func_8003CD98(gPlayerOneCopy, camera2, 0, 1);
-            func_8003CD98(gPlayerTwo, camera2, 1, 1);
-            func_8003CD98(gPlayerThree, camera2, 2, 1);
-            func_8003CD98(gPlayerFour, camera2, 3, 1);
-            func_8003CD98(gPlayerFive, camera2, 4, 1);
-            func_8003CD98(gPlayerSix, camera2, 5, 1);
-            func_8003CD98(gPlayerSeven, camera2, 6, 1);
-            func_8003CD98(gPlayerEight, camera2, 7, 1);
+            for (int i = 0; i < NUM_PLAYERS; i++) {
+                func_8003CD98(&gPlayers[i], camera1, i, 0);
+                func_8003CD98(&gPlayers[i], camera2, i, 1);
```

Comment:
Order might matter. This isn't functionally equiv. And depending on what this function do could result in issues.